### PR TITLE
[stable/weave-scope] Add priorityClassName for DaemonSet

### PR DESCRIPTION
**Pull request by titilambert**
#### Is this a new chart
No

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/weave-scope/README.md [hunk 1]
```diff
@@ -87,6 +87,8 @@ The **agent** section controls how the Weave Scope node agent pods are installed
 | **dockerBridge** | the name of the Docker bridge interface | `docker0` |
 | **scopeFrontendAddr** | the host:port of a Scope frontend to send data to -- this is only needed in cases where the frontend is deployed separately from the agent (e.g. an install outside the cluster or a pre-existing install inside it) | |
 | **probeToken** | the token used to connect to Weave Cloud -- this is not needed for connecting to non-cloud Scope frontends | |
+| **probeToken** | the token used to connect to Weave Cloud -- this is not needed for connecting to non-cloud Scope frontends | |
```
Comment (djsly) [inline stable/weave-scope/README.md]: duplicated line

**Review by djsly — CHANGES_REQUESTED** [review]

**Comment by djsly** [comment]
@fbarl  @dholbach 

Could one of you two review and accept ?

Thanks!

File: stable/weave-scope/README.md (hunk 1 above)
Comment (titilambert) [reply stable/weave-scope/README.md]: done

**Comment by titilambert** [comment]
@fbarl @dholbach Any news on this PR ?

**Comment by djsly** [comment]
@bboreham could you also help with this request

**Comment by djsly** [comment]
thanks a lot @davidkarlsen , really appreciate

**Pull request merged**